imresize.py

```diff
@@ -108,6 +108,8 @@ def imresize(I, scalar_scale=None, method='bicubic', output_shape=None, mode="ve
         
     kernel_width = 4.0
     # Fill scale and output_size
+    if scalar_scale is not None and output_shape is not None:
+        raise ValueError('either scalar_scale OR output_shape should be defined')
```

Comment:
First of all, the logic here should be 
`scalar_scale is None and output_shape is None`
Second, it is already done by the line 119:
`raise ValueError('scalar_scale OR output_shape should be defined!')`

Could you please remove this two lines from pull request?

Reply:
Hi @fatheral, apologies for a long wait from my side. 

Yes, having the line `scalar_scale is None and output_shape is None` would also be nice, but I in fact meant the line `scalar_scale is not None and output_shape is not None` to alleviate the case when both parameters are specified, say, by mistake of the user, which leads to an ambiguous situation. Without this line, the if-else condition `if scalar_scale is not None:` will be satisfied first and its code block will be executed, so, if unaware of this behavior and misusing the interface of your script by providing both parameters at the same time, the user might end up with an unexpected result. Keeping this line would therefore notify them of that mistake instantly. I do not think this is a probable scenario, but I thought it would make sense to nip it in the bud anyway.

If you do believe this should be the desired behavior of your script, please confirm this, so I could remove those two lines from the commit. Thank you!

Reply:
Thanks for the clarification. Now I see, it is reasonable. Thank you for the pull request!

Reply:
Hi @fatheral, thanks!